### PR TITLE
Delete a photo

### PR DESCRIPTION
**Pull request by MichiBerger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,39 @@
-# Getting Started with Create React App
+# LittleSunshine
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+LittleSunshine makes it easy to keep the best phrases of your kids and save them as memory. You can simple add a phrase and combine it with a nice picture. The keep track of your favorite phrases you can bookmark them or if you get rid of a phrase you can just delete it! With LittleSunshine you´ll never forget the best memories with your kid!
```
Comment (ttaannjjaa) [inline README.md]: Hi Michi, did you mean "  To keep track of.... then there is a little typo

File: README.md [hunk 2]
```diff
@@ -1,70 +1,39 @@
...
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+You can find the hosted version on Vercel: [LittleSunshine - Vercel](https://capstone-project-ptsp735eh-michiberger.vercel.app/)
+Important: This demo is optimized for mobile screen.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Tech Stack
 
-### `npm test`
+- Create React App
```
Comment (ttaannjjaa) [inline README.md:18]: just as reminder - recommended in the code review last week were links (you can find some e.g. in my Readme)

File: public/index.html [hunk 3]
```diff
@@ -7,9 +7,9 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="LittleSunshine makes it easy to keep the best phrases of your kids and save them as memory"
     />
-    <title>Capstone Project - Name muss noch vergeben werden</title>
+    <title>LittleSunshine | Keep the best phrases of you kid!</title>
```
Comment (ttaannjjaa) [inline public/index.html]: typo I think: your

File: src/components/ModalDeleteMessage.stories.js [hunk 4]
```diff
@@ -11,6 +11,6 @@ export const Default = Template.bind({});
 Default.args = {
   deleteText: 'Löschen',
   cancelText: 'Abbrechen',
-  messageTitle: '',
-  messageText: '',
+  messageTitle: 'Spruch löschen',
+  messageText: 'Möchtest du wirklich löschen',
```
Comment (ttaannjjaa) [inline src/components/ModalDeleteMessage.stories.js:15]: maybe you want to add an  ? at the end of the question

**Review by ttaannjjaa — APPROVED** [review]
Great work, congratulations to the successful picture upload - only comments because of some little typos

File: README.md (hunk 1 above)
Comment (MichiBerger) [reply README.md]: haha thx ;)

File: public/index.html (hunk 3 above)
Comment (MichiBerger) [reply public/index.html]: thx

**Pull request merged**